st2debug/tests/integration/fixtures/submit-debug-info.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Similar comment to the st2doc change. Can you remove everything that's Plexxi specific from this file.